data/es.properties

```diff
@@ -1,3 +1,3 @@
-app_title = Aplicación vacía privilegiada
+app-title = Aplicación vacía privilegiada
```

Comment:
Ok, so I will be lazy :-)

I've looked at sms, and they are using this form very consistently. You know who to complain to ;-)